### ISSUE TITLE
Preserve layout

### ISSUE THREAD

**Issue by mgrazebrook** [tesseract-ocr/tesseract#951]
Would it be possible to preserve layout? I'm thinking of something akin to:

pdftotext -layout

I'm working with images of tabular computer printouts. Although the OCR function is impressive, without the layout, the result is largely unusable. If text stayed roughly in its columns, it would be readable.

**Comment by mgrazebrook** [comment]
I wasn't sure if --psm 5 might have been intended to be helpful but it produced junk: could it be broken? All the other psm options I tried produced something reasonable.

**Comment by Shreeshrii** [comment]
Have you looked at tsv and hocr ouputs?

**Comment by mgrazebrook** [comment]
I have now - they aren't mentioned on the default usage, so I didn't know about them. For what I want to do, the tsv fomat may work well. I'm not sure what some of the columns mean, but the coordinates are clear.

Your suggestion lets me develop a solution for my problem better than my original suggestion. In particular, it lets me make my own decisions about text which doesn't align well such as superscripts or font changes within a line.

However I think my change request as originally stated would be useful. If you try pdftotext with/out the -layout option on something like a bank statement you'll see what I mean! 

To summarise: 

Thanks, you solved my problem. My proposal would still be useful. The --help option should document tsv/hocr (a new change request)?. Thanks for an amazing tool.

**Comment by zdenop** [comment]
--help option does not document pdf neither. So there is no difference between pdf, tsv and hocr ;-)

**Comment by sirius0503** [comment]
> Have you looked at tsv and hocr ouputs?

Hi @Shreeshrii , can you explain how I can use the tsv and hocr format for maintaining the layout.

**Comment by zdenop** [comment]
Closing as there is no input from the original issue reporter
